test/activerecord_spanner_mock_server/aborted_transaction_test.rb

```diff
@@ -8,120 +8,122 @@
 
 require_relative "./base_spanner_mock_server_test"
 
-class AbortedTransactionTest < BaseSpannerMockServerTest
-  def test_read_write_transaction_without_abort_does_not_retry
-    register_insert_singer_result
-
-    ActiveRecord::Base.transaction do
-      Singer.create(first_name: "Dave", last_name: "Allison")
+module MockServerTests
```

Comment:
Added module names to the different tests to prevent clashes for the models with the same class names.